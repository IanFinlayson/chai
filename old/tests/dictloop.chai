def main():
    let yummyness = {"bbq sauce": 8, "salsa": 7, "ketchup": 3, "mustard": 6, "mayo": 7}
    for thing in yummyness:
        print(thing)



