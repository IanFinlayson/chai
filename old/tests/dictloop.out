("mustard", 6)
("bbq sauce", 8)
("mayo", 7)
("salsa", 7)
("ketchup", 3)
